subprojects/shipkit/src/test/groovy/org/shipkit/internal/gradle/versionupgrade/FindOpenPullRequestTest.groovy

```diff
@@ -0,0 +1,50 @@
+package org.shipkit.internal.gradle.versionupgrade
+
+import org.shipkit.internal.util.GitHubApi
+import spock.lang.Specification
+
+class FindOpenPullRequestTest extends Specification {
+
+    FindOpenPullRequest findOpenPullRequest = new FindOpenPullRequest()
+
+    def "should return null if response is empty"() {
+        given:
+        def gitHubApi = Mock(GitHubApi)
+        def findOpenPullRequestTask = Mock(FindOpenPullRequestTask)
```

Comment:
Hint: to simplify tests, you can pass repo and regex to the overloaded "findOpenPullRequest" method, instead of passing the task instance.

Reply:
Good idea, thanks!